README.md

```diff
@@ -83,6 +83,7 @@ With the `theme` field, you can define a custom theme configuration for the Arwe
 | `accent`         | `RGBObject`                        | RGB accent color for the UI                                            |
 | `titleHighlight` | `RGBObject`                        | RGB accent color for the subscreen titles (like the connection screen) |
 | `radius`         | `"default"`, `"minimal"`, `"none"` | Border radius level used throughout the Kit UI                         |
+| `font`           | `Font`                             | Including font family used throughout the Kit UI                       |
```

Comment:
Include a basic example on what does the `Font` interface you defined so people know how to use the font object in the theme

Reply:
done!